js/saveFormData.js

```diff
@@ -0,0 +1,15 @@
+const contactUsForm = document.forms[0];
+const formDetails = {};
+if (window.localStorage.getItem('data')) {
```

Comment:
- [ ] Please take note that when the user changes the content of any input field in the form the data object is updated and it is saved to the local storage. This is where the onChange() method on the input field comes in (in my opinion 😄)

Reply:
@vickymarz mmm again it is the same problem just will adjust my event listener to work with any change not just submitting it

Reply:
Exactly. That's what you are to do